packages/analytics/metric-cards/src/components/display/MetricsCard.vue

```diff
@@ -183,6 +186,10 @@ const props = defineProps({
     required: false,
     default: () => 'lg',
   },
+  titleTag: {
```

Comment:
Could you explain more about the use case behind making this a property?  I'd expect we'd want all metric cards to have the same title tag...

Reply:
That is the case most of the time, however if metrics card is seeded inside some other widget that already has an h2 heading - it should be h3 to not break the hierarchy - that's when this prop comes in handy. Regardless of what tag is used - styling for the title remains consistent though